unit_tests/log_transfer/CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ add_dependencies(test_prior_sendrecv
 # 
 
 set(TEST_LOG_PAGE_FETCHER_SOURCES
-  ${TRANSACTION_DIR}/log_page_fetcher.cpp
+  ${TRANSACTION_DIR}/async_page_fetcher.cpp
```

Comment:
By converting from log_page_fetcher to async_page_fetcher, the file becomes part of storage module. Move to ${STORAGE_DIR}